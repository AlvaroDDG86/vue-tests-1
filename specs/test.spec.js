import { mount, shallowMount } from '@vue/test-utils'

import TestComponent from '@/test.vue'
import List from '@/list.vue'
test('Mount the test component', () => {
    const wrapper = mount(TestComponent, {
        propsData: {
            value: 'Test'
        }
    })
    expect(wrapper.html()).toMatchSnapshot()
})

test('Mount the list component', () => {
    const wrapper = shallowMount(List)
    expect(wrapper.html()).toMatchSnapshot()
})
