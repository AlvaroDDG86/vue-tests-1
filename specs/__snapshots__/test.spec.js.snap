// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Mount the component 1`] = `
<div>
  <h1>Hello Test</h1>
</div>
`;
